src/System.ComponentModel.Annotations/pkg/System.ComponentModel.Annotations.pkgproj

```diff
@@ -1,26 +1,27 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.ComponentModel.Annotations.csproj">
```

Comment:
This should not have been deleted. This is causing your build failure.  Add it back and make supportedframework `netcoreapp3.0;$(AllXamarinFrameworks)`.  The project reference includes the reference assembly in your package, supported framework indicates which target frameworks you expect that reference to apply to.

Reply:
Oops, I didn't notice this was the one for ref. Indeed that fixes it.